### ISSUE TITLE
Reduction of frame number after lock screen

### ISSUE THREAD

**Issue by Dreamcast2** [Cpasjuste/psnes#27]
Test game: Super Metroid
System: 5.1+ SXOS
Enter the game is 60 FPS at full speed，Lock screen，turned on, enter the game again，it will drop to about 45 FPS.
Troublesome question, hope to fix it, thank you.
Temporary solution:  save state, exit game, load game, load state.

**Comment by mcamacho97** [comment]
Yes I have the same bug in any game, I hope that you can fix it too.
